stream_alert_cli/helpers.py

```diff
@@ -34,6 +34,41 @@
 from stream_alert.rule_processor.firehose import StreamAlertFirehose
 
 
+SCHEMA_TYPE_LOOKUP = {
+    bool: 'boolean',
+    float: 'float',
+    int: 'integer',
+    str: 'string',
+    dict: dict(),
+    list: list()
+}
+
+
+def record_to_schema(record, recursive=False):
```

Comment:
We're doing 2 layers of translation: The schema is converted here to 'boolean', 'float', etc, and then again in a downstream helper function to the underlying Hive types:

https://github.com/airbnb/streamalert/blob/master/stream_alert_cli/athena/helpers.py#L18-L25

Is there any way to skip this intermediate representation?

Reply:
@austinbyers can you clarify what you mean by `intermediate representation`? do you mean combine them into one? I added the record_to_schema because it could be handy elsewhere (for example: we had discussed potentially auto-generating schema suggestions for users if there is ever a 'failed parse')

Reply:
Ah, `record_to_schema` is expecting the same schema type as we use in `conf/logs.json`. That makes more sense, sorry I missed that. Can you add a comment somewhere clarifying that the `schema` being returned here is compatible with the log schema?

Reply:
yes sorry for the ambiguity/confusion :) I will add a comment to clarify - thanks for flagging!

Reply:
added the requested comment!